README.md

```diff
@@ -0,0 +1,467 @@
+  * [1. Set up the RDF4J graph db](#1-set-up-the-rdf4j-graph-db)
+    * [Docker compose config for the graph db container](#docker-compose-config-for-the-graph-db-container)
+    * [Initialization of the graph db repository](#initialization-of-the-graph-db-repository)
+    * [Configuration of the LDI Orchestrator](#configuration-of-the-ldi-orchestrator)
+  * [2. Set up the TRIFID subject pages](#2-set-up-the-trifid-subject-pages)
+    * [Custom Dockerfile](#custom-dockerfile)
+    * [Configuration template](#configuration-template)
+    * [Dataset base url](#dataset-base-url)
+    * [Middlewares](#middlewares)
+      * [a. Rewrite](#a-rewrite)
+      * [b. Entity Renderer](#b-entity-renderer)
+      * [c. Welcome](#c-welcome)
+      * [d. Sparql-handler](#d-sparql-handler)
+      * [Middleware node modules](#middleware-node-modules)
+    * [Server port](#server-port)
+    * [Add Trifid to docker compose](#add-trifid-to-docker-compose)
+<!-- TOC -->
+
+[//]: # (@formatter:on)
+
+## What are subject pages?
+
+Subject pages are web pages that can display the information of an entity's that is part of a linked data event stream (
+LDES).
+
+# Tutorial: how to set up subject pages?
+
+In this tutorial, we will set up subject pages for the LDES
+of [GIPOD](https://www.vlaanderen.be/digitaal-vlaanderen/onze-oplossingen/generiek-informatieplatform-openbaar-domein-gipod).
+To set these pages up, a graph db is required. Any graph db can be used, but in this tutorial, we are
+using [RDF4J from Eclips](https://rdf4j.org/). If you already have a graph db, or you want to use another one, you can
+directly go to [step 2](#2-set-up-the-trifid-subject-pages).
+
+## 1. Set up the RDF4J graph db
+
+To set up the graph db, we will be using
+the [LDI Orchestrator (LDIO)](https://informatievlaanderen.github.io/VSDS-Linked-Data-Interactions/ldio/index),
+where we will use
+the [LDES Client](https://informatievlaanderen.github.io/VSDS-Linked-Data-Interactions/ldio/ldio-inputs/ldio-ldes-client)
+as input component
+and
+the [Repository Materialiser](https://informatievlaanderen.github.io/VSDS-Linked-Data-Interactions/ldio/ldio-outputs/ldio-repository-materialiser)
+as output component.
+
+### Docker compose config for the graph db container
+
+Create a folder where all files for setting up the
+
+```shell
+mkdir subject-pages
+cd subject-pages
+```
+
+Add a `docker-compose.yml` file to this directory and add the following config to that file
+
+```yaml
+version: '3.5'
+
+services:
+  graph-db:
+    image: eclipse/rdf4j-workbench:latest
+    container_name: rdf4j-server
+    environment:
+      - "JAVA_OPTS=-Xms1g -Xmx4g"
+    ports:
+      - "8080:8080"
+    networks:
+      - subject-pages-network
+
+networks:
+  subject-pages-network:
+    driver: bridge
+    name: subject-pages-network
+
+```
+
+### Initialization of the graph db repository
+
+We want our repository to be initialized when we spin up our containers. Therefor, we use a simple ubuntu container that
```

Comment:
Why do we need an ubuntu container for a simple curl command? I thought windows has curl support as well?